internal/cli/internal/command/open/open.go

```diff
@@ -0,0 +1,69 @@
+package open
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/skratchdot/open-golang/open"
+	"github.com/spf13/cobra"
+
+	"github.com/superfly/flyctl/internal/buildinfo"
+	"github.com/superfly/flyctl/internal/cli/internal/app"
+	"github.com/superfly/flyctl/internal/cli/internal/command"
+	"github.com/superfly/flyctl/internal/cli/internal/flag"
+	"github.com/superfly/flyctl/internal/client"
+	"github.com/superfly/flyctl/pkg/iostreams"
```

Comment:
I'm so sorry I didn't spot this: `pkg` imports should be listed after 3rd party ones, in their own section.

You can see an example of this [here](https://github.com/superfly/flyctl/blob/c98d6cadd9d491abfcf7e8f3060d79dee19ad82b/internal/cli/internal/command/apps/create.go#L3)